Detectors/Passive/src/FrameStructure.cxx

```diff
@@ -429,15 +429,19 @@ void FrameStructure::ConstructGeometry()
   //
   TGeoPgon* shB77A = new TGeoPgon(0., 360., 18, 2);
   shB77A->SetName("shB77A");
-  shB77A->DefineSection(0, -zE, 280., 423.7);
-  shB77A->DefineSection(1, zE, 280., 423.7);
+  shB77A->DefineSection(0, -zE, 280., 422.15);
+  shB77A->DefineSection(1, zE, 280., 422.15);
   TGeoBBox* shB77B = new TGeoBBox(3.42, 2., 375.5);
   shB77B->SetName("shB77B");
   TGeoTranslation* trB77A = new TGeoTranslation("trB77A", +283.32, 0., 0.);
   TGeoTranslation* trB77B = new TGeoTranslation("trB77B", -283.32, 0., 0.);
   trB77A->RegisterYourself();
   trB77B->RegisterYourself();
-  TGeoCompositeShape* shB77 = new TGeoCompositeShape("shB77", "shB77A+shB77B:trB77A+shB77B:trB77B");
+  TGeoPgon* shB77C = new TGeoPgon(60., 360., 18, 2);
+  shB77C->SetName("shB77C");
+  shB77C->DefineSection(0, -zE, 422.15, 423.70);
+  shB77C->DefineSection(1, zE, 422.15, 423.70);
```

Comment:
This looks like a tube. We should avoid using complex solids when they can be represented as simple entities for performance reasons.

Reply:
The boundary between FRAME and EMC is a polygon. If I change the shape to a TubeSeg
I get either extrusions of elements in FRAME or overlaps with EMC.

Reply:
Ok. From the looks of it this shouldn't make a difference. But let's leave it like this.